WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -348,9 +348,9 @@ extension OrderDetailsViewModel {
         let providerName = tracking.trackingProvider ?? ""
 
         ServiceLocator.analytics.track(.orderTrackingDelete, withProperties: ["id": orderID,
-                                                                         "status": statusKey,
-                                                                         "carrier": providerName,
-                                                                         "source": "order_detail"])
+                                                                              "status": statusKey,
```

Comment:
this might require the raw value from the enum? otherwise, I get a crash in Tracks:

```
[error] error: SQLCore dispatchRequest: exception handling request: <NSSQLSaveChangesRequestContext: 0x600002304840> , The data couldn’t be written because it isn’t in the correct format. with userInfo of {
    NSUnderlyingError = "Error Domain=NSCocoaErrorDomain Code=4866 \"The data couldn\U2019t be written because it isn\U2019t in the correct format.\" UserInfo={NSUnderlyingError=0x600001b48330 {Error Domain=NSCocoaErrorDomain Code=4864 \"This decoder will only decode classes that adopt NSSecureCoding. Class '__SwiftValue' does not adopt it.\" UserInfo={NSDebugDescription=This decoder will only decode classes that adopt NSSecureCoding. Class '__SwiftValue' does not adopt it.}}}";
}
```

```suggestion
                                                                              "status": statusKey.rawValue,
```